### PR TITLE
Fixes #469 - Improve the styling of Signup page

### LINKED ISSUE #469: Improve style of Signup form
**Actual Behaviour**

The borders of the Signup form fields aren't styled, and not enough spacing between the form fields and the text.

**Expected Behaviour**

Style to borders to be consistent with the Login Form. Add some spacing between the form fields and the text

**Screenshots of the issue**

screenshot 18

**Would you like to work on the issue?**

Yes I would like to work on the issue

### PR DESCRIPTION
**Pull request by rossoskull**
Fixes #469 

Changes: 
- Improve Signup page input fields border style.
- Add spacing between text and input fields.

Surge Deployment Link: https://pr-470-fossasia-susi-accounts.surge.sh

Screenshots for the change:

screenshot 21

@anshumanv @arundhati24 @PrP-11, Please review this PR.

### CODE REVIEW

**Comment by pg07codes** [comment]
@rossoskull The layout of the complete page has to be fixed and not only the styling . please review the comments on PR #469

**Comment by rossoskull** [comment]
@pg07codes The Issue #469 was about styling the borders and adding spacing between the form fields and the text. That's what I have done in this PR.

File: src/components/Auth/SignUp/SignUp.react.js [hunk 1]
```diff
@@ -185,11 +185,13 @@ export default class SignUp extends Component {
...
   render() {
     const fieldStyle = {
-      width: '94%',
+      width: '100%',
       height: '30px',
       marginBottom: '3%',
+      border: '1px solid #ced4da',
       borderRadius: '5px',
-      paddingLeft: '20px',
+      paddingLeft: '10px',
```
Comment (anshumanv) [inline src/components/Auth/SignUp/SignUp.react.js]: Make it 5px as done on the login fields. Rest looks good.

**Comment by rossoskull** [comment]
@anshumanv Done. Thanks!

**Comment by rossoskull** [comment]
Thanks! 😃

**Pull request merged**